ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_random.py

```diff
@@ -1,12 +1,52 @@
 # global
 from hypothesis import strategies as st
+import ivy
```

Comment:
I think if you change how the argument of `input_dtypes` in `handle_frontend_test`  as I mentioned below, you can get rid of this import